src/content/structured/components/chips/code.mdx

```diff
@@ -111,27 +167,81 @@ export const snippetsDefault = [
 export const snippetsOutline = [
   {
     language: "Web component",
-    snippet: `<ic-chip label="Small" size="small" appearance="outline">
-  <svg slot="icon" ... />
-</ic-chip>
-<ic-chip label="Default" appearance="outline">
-  <svg slot="icon" ... />
-</ic-chip>
-<ic-chip label="Large" size="large" appearance="outline">
-  <svg slot="icon" ... />
-</ic-chip>`,
+    snippet: `  <ic-chip label="Small" size="small" appearance="outline">
```

Comment:
Think the gap here needs removing as it makes the tabbing of the snippet look odd